docs/en/guides/README.md

```diff
@@ -119,11 +119,12 @@ You would most likely have to add a new source and scope. To learn how to do thi
 
 As one of the Top Level Projects of The Apache Software Foundation (ASF), SkyWalking must follow the [ASF 3RD PARTY LICENSE POLICY](https://apache.org/legal/resolved.html). So if you're adding new dependencies to the project, you should make sure that the new dependencies would not break the policy, and add their LICENSE and NOTICE to the project.
 
-We have a [simple script](../../../tools/dependencies/check-LICENSE.sh) to help you make sure that you haven't missed out any new dependencies:
-- Build a distribution package and unzip/untar it to folder `dist`.
-- Run the script in the root directory. It will print out all new dependencies.
-- Check the LICENSE and NOTICE of those dependencies to make sure that they can be included in an ASF project. Add them to the `apm-dist/release-docs/{LICENSE,NOTICE}` file.
-- Add the names of these dependencies to the `tools/dependencies/known-oap-backend-dependencies.txt` file (**in alphabetical order**). `check-LICENSE.sh` should pass in the next run.
+We use [license-eye](https://github.com/apache/skywalking-eyes) to help you make sure that you haven't missed out any new dependencies:
+- Install `license-eye` according to [the doc](https://github.com/apache/skywalking-eyes#usage).
+- Run `license-eye dependency resolve --summary ./dist-material/release-docs/LICENSE.tpl` in the root directory of this project.
+- Check the modified lines in `./dist-material/release-docs/LICENSE` (via command `git diff -U0 ./dist-material/release-docs/LICENSE`) and
+  check whether the new dependencies' licenses are compatible with Apache 2.0.
+- Add the new dependencies' notice (if any) to `./dist-material/release-docs/NOTICE`, copy their license file to `./dist-material/release-docs/licenses` if they are not standard Apache 2.0 license.
```

Comment:
```suggestion
- Add the new dependencies' notice files (if any) to `./dist-material/release-docs/NOTICE` if they are Apache 2.0 license. Copy their license files to `./dist-material/release-docs/licenses` if they are not standard Apache 2.0 license.
```

Reply:
NOTICE file only matters for Apache 2.0

Reply:
> Suggested change 
> - Add the new dependencies' notice (if any) to `./dist-material/release-docs/NOTICE`, copy their license file to `./dist-material/release-docs/licenses` if they are not standard Apache 2.0 license.
> - Add the new dependencies' notice files (if any) to `./dist-material/release-docs/NOTICE` if they are Apache 2.0 license. Copy their license files to `./dist-material/release-docs/licenses` if they are not standard Apache 2.0 license.

> Add the new dependencies' notice files (if any) to `./dist-material/release-docs/NOTICE` if they are Apache 2.0 license.

Do you think this is correct? We only need to copy the notice if they are apache 2.0 license?

Reply:
Yes, I am sure. Notice file is mentioned in Apache 2.0 License. And most other licenses are not mentioning this. At least most widely used licenses are not, such as MIT, BSD 2/3

Reply:
No, we should not.

Let's take a look at the full content of MIT, https://opensource.org/licenses/MIT.
The whole text, it doesn't mention anything about the NOTICE file. 

Then let's see the APLv2, https://github.com/apache/skywalking/blob/0ac9874e79e1826e352f2c4a05c93aed7234700e/LICENSE#L106
> (d) If the Work includes a "NOTICE" text file as part of its distribution, ...

APLv2 declares the NOTICE file is actually a part of the LICENSE file, which is why we need to copy.

Reply:
I remember there was a discussion between Justin and me when DolphinScheduler was still incubating. But I can't find the thread anymore. But the contents of the license should be official guidance, and I can remember this principle due to this.